src/status_im/ui2/screens/chat/actions.cljs

```diff
@@ -40,14 +41,37 @@
 (defn unmute-chat-action [chat-id]
   (hide-sheet-and-dispatch [::chat.models/mute-chat-toggled chat-id false]))
 
-(defn clear-history-action [chat-id]
-  (hide-sheet-and-dispatch [:chat.ui/clear-history-pressed chat-id]))
-
-(defn delete-chat-action [chat-id]
-  (hide-sheet-and-dispatch [:chat.ui/remove-chat-pressed chat-id]))
-
-(defn leave-group-action [chat-id]
-  (hide-sheet-and-dispatch [:group-chats.ui/leave-chat-pressed chat-id]))
+(defn clear-history-action [{:keys [chat-id] :as item}]
+  (hide-sheet-and-dispatch [:bottom-sheet/show-sheet
+                            {:content #(common/alert {:title       (i18n/label :t/clear-history?)
```

Comment:
FYI, nitpick: It's more common in Clojure to reverse the usage of the `#()` reader macro and `(fn`) for readability purposes. The rationale is that the reader macro `#()` is often used for quick, single line functions, otherwise readers need to scan multiple lines to find the `%` sign.

```clojure
(fn []
  (common/alert {:title       (i18n/label :t/clear-history?)
                 :description (i18n/label :t/clear-history-confirmation-content)
                 :context     item
                 :button-text (i18n/label :t/clear-history)
                 :on-press    #(hide-sheet-and-dispatch [:chat.ui/clear-history chat-id])}))
```